Source/WebKit/WebProcess/WebCoreSupport/WebPopupMenu.cpp

```diff
@@ -60,66 +60,67 @@ void WebPopupMenu::disconnectClient()
 
 void WebPopupMenu::didChangeSelectedIndex(int newIndex)
 {
-    if (!m_popupClient)
-        return;
-
-    m_popupClient->popupDidHide();
-    if (newIndex >= 0)
-        m_popupClient->valueChanged(newIndex);
+    if (CheckedPtr popupClient = m_popupClient) {
```

Comment:
We prefer early returns in WebKit in general. We sometimes make exceptions for one liners but this looks like something that should have kept the old style.